src/commands/init/ai-rules.ts

```diff
@@ -0,0 +1,447 @@
+  const configPath = resolve(projectPath, '.vscode', 'mcp.json')
+
+  try {
+    // Create .vscode directory if it doesn't exist
+    await fs.mkdir(resolve(projectPath, '.vscode'), { recursive: true })
+
+    // Write or update mcp.json
+    let existingConfig: Record<string, unknown> = {}
+    try {
+      const existingContent = await fs.readFile(configPath, 'utf-8')
+      existingConfig = JSON.parse(existingContent) as Record<string, unknown>
+    } catch {
+      // File doesn't exist or is invalid JSON
+    }
+
+    const mcpConfig = JSON.parse(config) as Record<string, unknown>
+    const updatedConfig = { ...existingConfig, ...mcpConfig }
+
+    await fs.writeFile(configPath, JSON.stringify(updatedConfig, null, 2), 'utf-8')
+    log(`${chalk.green('✅')} VS Code MCP configuration saved to ${chalk.cyan('.vscode/mcp.json')}`)
+  } catch (error) {
+    throw new Error(`Failed to configure VS Code MCP: ${error instanceof Error ? error.message : 'Unknown error'}`)
+  }
+}
+
+// Cursor specific MCP configuration
+const configureMcpForCursor = async (config: string, projectPath: string): Promise<void> => {
+  const configPath = resolve(projectPath, '.cursor', 'mcp.json')
+
+  try {
+    await fs.mkdir(resolve(projectPath, '.cursor'), { recursive: true })
+    await fs.writeFile(configPath, config, 'utf-8')
+    log(`${chalk.green('✅')} Cursor MCP configuration saved to ${chalk.cyan('.cursor/mcp.json')}`)
+  } catch (error) {
+    throw new Error(`Failed to configure Cursor MCP: ${error instanceof Error ? error.message : 'Unknown error'}`)
+  }
+}
+
+// Windsurf specific MCP configuration
+const configureMcpForWindsurf = async (config: string, _projectPath: string): Promise<void> => {
+  const { homedir } = await import('node:os')
+  const configPath = resolve(homedir(), '.codeium', 'windsurf', 'mcp_config.json')
+
+  try {
+    // Create .codeium/windsurf directory if it doesn't exist
+    await fs.mkdir(resolve(homedir(), '.codeium', 'windsurf'), { recursive: true })
+
+    // Read existing config or create new one
+    let existingConfig: Record<string, unknown> = {}
+    try {
+      const existingContent = await fs.readFile(configPath, 'utf-8')
+      existingConfig = JSON.parse(existingContent) as Record<string, unknown>
+    } catch {
+      // File doesn't exist or is invalid JSON
+    }
+
+    const mcpConfig = JSON.parse(config) as Record<string, unknown>
+
+    // Merge mcpServers from both configs
+    const existingServers = (existingConfig.mcpServers as Record<string, unknown> | undefined) ?? {}
+    const newServers = (mcpConfig.mcpServers as Record<string, unknown> | undefined) ?? {}
+
+    const updatedConfig = {
+      ...existingConfig,
+      mcpServers: {
+        ...existingServers,
+        ...newServers,
+      },
+    }
+
+    await fs.writeFile(configPath, JSON.stringify(updatedConfig, null, 2), 'utf-8')
+    log(`${chalk.green('✅')} Windsurf MCP configuration saved to global config`)
```

Comment:
What does "global config" refer to here?

Reply:
means windsurf mcp config file